src/MongoDB/ReadConcern.c

```diff
@@ -101,6 +101,16 @@ PHP_METHOD(ReadConcern, getLevel)
 }
 /* }}} */
 
+/* {{{ proto array ReadConcern::bsonSerialize()
+   */
+PHP_METHOD(ReadConcern, bsonSerialize)
+{
+	const mongoc_read_concern_t *read_concern = phongo_read_concern_from_zval(getThis() TSRMLS_CC);
```

Comment:
It appears that `-Wno-unused-parameter` was introduced in https://github.com/mongodb/mongo-php-driver/commit/e3e3c9d552e52484f504c31a7cb0f496b437f76a#diff-788d457a20b110cc38e571dec9ddc68cR126, builds have been ignoring unused variables since then.